builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
Here is scenario that breaks down if we keep it and honour it for `autoplay`:

* Let's say we want **muted autoplay** on **desktop and mobile**, but users should be able to **unmute the video as well**. (Essentially the most common use-case)

What's the markup for this scenario?
a) `<amp-video muted autoplay>`
b) `<amp-video autoplay>`

Neither will do what we want because:
a) will give us unmuted autoplay, but user won't be able to unmute in iOS
b) will give us unmuted autoplay on mobile but not on desktop.

If we want to support unmuted autoplay on desktop, we should provide a new attribute maybe `autoplay-with-sound` 

If we keep `muted` but not honour it for autoplay, then we are back to `muted` attribute being mostly useless, because all it can do is decide if tapping a video (autoplay or not) plays the video with sound or not (with the big caveat that iOS users won't be able to unmute it because there is no mute/unmute control in iOS)